src/graph/network.cc

```diff
@@ -105,18 +82,18 @@ DGL_REGISTER_GLOBAL("network._CAPI_ReceiverRecvSubgraph")
     if (size <= 0) {
       LOG(ERROR) << "Receive error: (size: " << size << ")";
     }
-    NodeFlow nf;
+    NodeFlow* nf = new NodeFlow();
```

Comment:
how is the memory free'd?

Reply:
@zheng-da 
I do the same allocation like this:

https://github.com/dmlc/dgl/blob/eb1acecd0d9057d116fa8c8687123600a5065557/src/graph/sampler.cc#L836

Reply:
How does the sampler free the memory?

Reply:
They are manually freed from Python code after instantiation of `NodeFlow` objects (`python/dgl/nodeflow.py:59`)